### PR TITLE
Add DTO for ES

### PR DESCRIPTION
**Pull request by Beacze**
Update type classes with missing values for DTO.

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/dto/ProcessDTO.java [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+        this.panelShown = panelShown;
+    }
+
+    /**
+     * Get information if task is selected.
+     * 
+     * @return true or false
+     */
+    public boolean isSelected() {
+        return (this.selected);
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/dto/ProcessDTO.java]: Why did you add the round braces?

File: Kitodo/src/main/java/org/kitodo/dto/TaskDTO.java [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+        this.panelShown = panelShown;
+    }
+
+    /**
+     * Get information if task is selected.
+     *
+     * @return true or false
+     */
+    public boolean isSelected() {
+        return (this.selected);
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/dto/TaskDTO.java]: Same here.

File: Kitodo/src/main/java/org/kitodo/dto/ProcessDTO.java (hunk 1 above)
Comment (Beacze) [inline Kitodo/src/main/java/org/kitodo/dto/ProcessDTO.java]: no idea :D

File: Kitodo/src/main/java/org/kitodo/dto/TaskDTO.java (hunk 2 above)
Comment (Beacze) [inline Kitodo/src/main/java/org/kitodo/dto/TaskDTO.java]: probably just copied from previous one
